reference/6/Microsoft.PowerShell.Utility/Export-Clixml.md

```diff
@@ -249,23 +273,29 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
+
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable,
+-InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose,
+-WarningAction, and -WarningVariable. For more information, see
+[about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
 
 ## INPUTS
 
 ### System.Management.Automation.PSObject
-You can pipe any object to **Export-Clixml**.
+
+You can pipe any object to `Export-Clixml`.
 
 ## OUTPUTS
 
 ### System.IO.FileInfo
-**Export-Clixml** creates a file that contains the XML.
+
+`Export-Clixml` creates a file that contains the XML.
 
 ## NOTES
 
 ## RELATED LINKS
```

Comment:
There were two preexisting external links in the documents. Should they be removed?

Reply:
Confirmed that the links are acceptable.